### ISSUE TITLE
nrf52 thingy target ble stack does not come back up after an erase command

### ISSUE THREAD

**Issue by jacobrosenthal** [apache/mynewt-core#795]
This is apparently not related to https://github.com/apache/mynewt-core/pull/789

```
$ newt target show split-thingy
targets/split-thingy
    app=@apache-mynewt-core/apps/blesplit
    bsp=@apache-mynewt-core/hw/bsp/nrf52-thingy
    build_profile=optimized
    loader=@apache-mynewt-core/apps/bleprph
    syscfg=CONSOLE_ECHO=0:CONSOLE_INPUT=0:CONSOLE_MAX_INPUT_LEN=0:CONSOLE_RTT=1:CONSOLE_UART=0:JLINK_DEBUG=1:LOG_LEVEL=0:SHELL_TASK=0
```

```
$ newt build split-thingy && newt create-image split-thingy 0.0.0.1 && newt load split-thingy 0.0.0.1
Building target targets/split-thingy
...
Loading app image into slot 2
Loading loader image into slot 1
$ newt create-image split-thingy 0.0.0.2
...
App image succesfully generated: /Users/jacobrosenthal/Downloads/chippd3/bin/targets/split-thingy/loader/apps/bleprph/bleprph.img
App image succesfully generated: /Users/jacobrosenthal/Downloads/chippd3/bin/targets/split-thingy/app/apps/blesplit/blesplit.img
$ ./test2.sh

**** Running: newtmgr -c nimble_bleprph image erase *****

Done

**** Running: newtmgr -c nimble_bleprph image list *****

Images:
 slot=0
    version: 0.0.0.1
    bootable: true
    flags: active confirmed
    hash: a5cc4e507442d75e130f9f2ae69b569f21eb9da3e74c50a57d6a4fded31c99e0
Split status: N/A (0)

**** Running: newtmgr -c nimble_bleprph image upload /Users/jacobrosenthal/Downloads/chippd3/bin/targets/split-thingy/loader/apps/bleprph/bleprph.img *****

 125.98 KiB / 126.16 KiB [==============================================================================================================]  99.85%Done

**** Running: newtmgr -c nimble_bleprph image list *****

Images:
 slot=0
    version: 0.0.0.1
    bootable: true
    flags: active confirmed
    hash: a5cc4e507442d75e130f9f2ae69b569f21eb9da3e74c50a57d6a4fded31c99e0
 slot=1
    version: 0.0.0.2
    bootable: true
    flags: 
    hash: 517d45dace367a948a771e329f9efd890ef4d367f11b3a44b5104aa5455c8a56
Split status: N/A (0)

**** Running: newtmgr -c nimble_bleprph image erase *****

Error: disconnected

**** Running: newtmgr -c nimble_bleprph image list *****

Error: Failed to connect to peer after 10s
```

logs
```
000001 [ts=7812ssb, mod=64 level=0] registered service 0x1800 with handle=1
000002 [ts=15624ssb, mod=64 level=0] registering characteristic 0x2a00 with def_handle=2 val_handle=3
000002 [ts=15624ssb, mod=64 level=0] registering characteristic 0x2a01 with def_handle=4 val_handle=5
000002 [ts=15624ssb, mod=64 level=0] registering characteristic 0x2a02 with def_handle=6 val_handle=7
000002 [ts=15624ssb, mod=64 level=0] registering characteristic 0x2a03 with def_handle=8 val_handle=9
000002 [ts=10x03 ocf=0x0063 len=8
000004 [ts=31248ssb, mod=4 level=0] 0x63 0x0c 0x08 0x00 0x00 0x80 0x00 0x00 0x00 0x00 0x00 
000004 [ts=31248ssb, mod=4 level=0] Command complete: cmd_pkts=1 ogf=0x3 ocf=0x63 status=0 
000004 [ts=31248ssb, mod=4 level=0] ble_hs_hci_cmd_send: ogf=0x08 ocf=0x0001 len=8
000004 [ts=31248ssb, mod=4 level=0] 0x01 0x20 0x08 0x7f 0x1a 0x0f 0x00 0x00 0x00 0x00 0x00 
0r0004 [ts=a659e-897e-45e1-b016-007107c96dd complete: cmd_pkts=1 ogf=0x8 ocf=0x1 status=0 
436ssb, mod=4 level=0] ble_hs_el=0] ble_hs_hci_cmd_send: ogf=0x08 ocf=0x0002 len=0
000004 [ts=31248ssb, mod=4 level=0] 0x02 0x20 0x00 
000005 [ts=39060ssb, mod=4 level=0] Command complete: cmd_pkts=1 ogf=0x8 ocf=0x2 status=0 
000005 [ts=39060ssb, mod=4 level=0] ble_hs_hci_cmd_send: ogf=0x08 ocf=0x0003 len=0
000005 [ts=39060ssb, mod=4 level=0] 0x03 0x20 0x00 
000005 [ts=39060ssb, mod=4 level=0] Command complete: cmd_pkts=1 ogf=0x8 ocf=0x3 status=0 
000005 [ts=39060ssb, mod=4 level=0] ble_hs_hci_cmd_send: ogf=0x04 ocf=0x0009 00006 [ts=46872ssb, mod=4 level=0] ble_hs_hci_cmd_send: ogf=0x08 ocf=0x0005 len=6
000006 [ts=46872ssb, mod=4 level=0] 0x05 0x20 0x06 0xed 0xf3 0x62 0x90 0xb6 0xd6 
000007 [ts=54684ssb, mod=4 level=0] Command complete: cmd_pkts=1 ogf=0x8 ocf=0x5 status=0 
000007 [ts=54684ssb, mod=4 level=0] ble_hs_hci_cmd_send: ogf=0x08 ocf=0x0007 len=0
000007 [ts=54684ssb, mod=4 level=0] 0x07 0x20 0x00 
000007 [ts=54684ssb, mod=4 level=0] Command complete: cmd_pkts=1 ogf=0x8 ocf=0x7 status=0 
b00007 [ts= 0xc0 0x53 0x1e 0xb8 0xef 0x8d 0xei_0xd_send: ogf=0x08 ocf=0x0008 len=32
000007 [ts=54684ssb, mod=4 level=0] 0x08 0x20 sb, mod=4 level=0] Command complete: cmd_pkts=1 ogf=0x8 ocf=0x27 status=0 
000006 [ts=46872ssb, mod=4 level=0] ble_hs_hci_cmd_send: ogf=0x08 ocf=0x004e len=8
000006 [ts=46872ssb, mod=4 level=0] 0x4e 0x20 0x08 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x01 
000006 [ts=46872ssb, mod=4 level=0] Command complete: cmd_pkts=1 ogf=vertise; disc_mode=2 adv_channel_map=0 own_addr_type=1 adv_filter_policy=0 adv_itvlmin=0 adv_itvl_max=0
000007 [ts=54684ssb, mod=4 level=0] ble_hs_hci_cmd_send: ogf=0x08 ocf=0x0006 len=15
000008 [ts=62496ssb, mod=4 level=0] 0x06 0x20 0x0f 0x30 0x00 0x60 0x00 0x00 0x01 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x07 0x00 
000008 [ts=62496ssb, mod=4 level=0] Command complete: cmd_pkts=1 ogf=0x8 ocf=0x6 status=0 
000008 [ts=62496ssb, mod=4 level=0] ble_hs_hci_cmd_send: ogf=0x08 ocf=0x000a len=1
000008 [ts=62496ssb, mod=4 level=0] 0x0a 0x20 0x01 0x01 
000008 [ts=62496ssb, mod=4 level=0] Command complete: cmd_pkts=1 ogf=0x8 ocf=0xa status=0 
000008 [ts=62496ssb, mod=4 level=0] Command complete: cmd_pkts=1 ogf=0x0 ocf=0x0
007947 [ts=62085932ssb, mod=4 level=0] LE connection complete. handle=1 role=1 paddrtype=0 addr=b8.e8.56.3.d3.ed local_rpa=0.0.0.0.0.0 peer_rpa=0.0.0.0.0.0 itvl=12 latency=0 spvn_tmo=200 mca=5
007947 [ts=62085932ssb, mod=64 level=1] connection established; status=0 handle=1 our_ota_addr_type=1 our_ota_addr=d6:b6:90:62:f3:ed our_id_addr_type=1 our_id_addr=d6:b6:90:62:f3:ed peer_ota_addr_type=0 peer_ota_addr=b8:e8:56:03:d3:ed peer_id_addr_type=0 peer_id_addr=b8:e8:56:03:d3:ed conn_itvl=12 conn_latency=0 supervision_timeout=200 encrypted=0 authenticated=0 bonded=0
007947 [ts=62085932ssb, mod=64 level=1] 
007947 [ts=62085932ssb, mod=4 level=0] ble_hs_hci_cmd_send: ogf=0x08 ocf=0x0016 len=2
007947 [ts=62085932ssb, mod=4 level=0] 0x16 0x20 0x02 0x01 0x00 
007947 [ts=62085932ssb, mod=4 level=0] Command Status: status=0 cmd_pkts=1 ocf=0x16 ogf=0x8
007948 [ts=62093744ssb, mod=4 level=0] 	Unknown LE event
007952 [ts=62124992ssb, mod=4 level=0] LE Remote Used Features. FAIL (status=26)
007954 [ts=62140616ssb, mod=4 level=0] ble_hs_hci_evt_acl_process(): conn_handle=1 pb=2 len=7 data=0x03 0x00 0x04 0x00 0x02 0x68 0x00 
007954 [ts=62140616ssb, mod=4 level=0] rxed att command: mtu req; conn=1 mtu=104
007954 [ts=62140616ssb, mod=4 level=0] txed att command: mtu rsp; conn=1 mtu=256
007954 [ts=62140616ssb, mod=4 level=0] host tx hci data; handle=1 length=7
007954 [ts=62140616ssb, mod=4 level=0] ble_hs_hci_acl_tx(): 0x01 0x00 0x07 0x00 0x03 0x00 0x04 0x00 0x03 0x00 0x01 
007954 [ts=62140616ssb, mod=64 level=1] mtu update event; conn_handle=1 cid=4 mtu=104
007958 [ts=62171864ssb, mod=4 level=0] ble_hs_hci_evt_acl_process(): conn_handle=1 pb=2 len=11 data=0x07 0x00 0x04 0x00 0x10 0x01 0x00 0xff 0xff 0x00 0x28 
007958 [ts=62171864ssb, mod=4 level=0] rxed att command: read group type req; conn=1 start_handle=0x0001 end_handle=0xffff
007958 [ts=62171864ssb, mod=4 level=0] txed att command: read group type rsp; conn=1 length=6
007958 [ts=62171864ssb, mod=4 level=0] host tx hci data; handle=1 length=24
007958 [ts=62171864ssb, mod=4 level=0] ble_hs_hci_acl_tx(): 0x01 0x00 0x18 0x00 0x14 0x00 0x04 0x00 0x11 0x06 0x01 0x00 0x0b 0x00 0x00 0x18 0x0c 0x00 0x0f 0x00 0x01 0x18 0x10 0x00 0x1c 0x00 0x11 0x18 
007959 [ts=62179676ssb, mod=4 level=0] Number of Completed Packets: num_handles=1
007959 [ts=62179676ssb, mod=4 level=0] handle:1 pkts:1
007962 [ts=62203112ssb, mod=4 level=0] ble_hs_hci_evt_acl_process(): conn_handle=1 pb=2 len=11 data=0x07 0x00 0x04 0x00 0x10 0x1d 0x00 0xff 0xff 0x00 0x28 
007962 [ts=62203112ssb, mod=4 level=0] rxed att command: read group type req; conn=1 start_handle=0x001d end_handle=0xffff
007962 [ts=62203112ssb, mod=4 level=0] txed att command: read group type rsp; conn=1 length=20
007962 [ts=62203112ssb, mod=4 level=0] host tx hci data; handle=1 length=46
007962 [ts=62203112ssb, mod=4 level=0] ble_hs_hci_acl_tx(): 0x01 0x00 0x2e 0x00 0x2a 0x00 0x04 0x00 0x11 0x14 0x1d 0x00 0x20 0x00 0x84 0xaa 0x60 0x74 0x52 0x8a 0x8b 0x86 0xd3 0x4c 0xb7 0x1d 0x1d 0xdc 0x53 0x8d 0x21 0x00 0xff 0xff 0x2d 0x71 0xa2 0x59 0xb4 0x58 0xc8 0x12 0x99 0x99 0x43 0x95 0x12 0x2f 0x46 0x59 
007963 [ts=62210924ssb, mod=4 level=0] Number of Completed Packets: num_handles=1
007963 [ts=62210924ssb, mod=4 level=0] handle:1 pkts:1
007966 [ts=62234360ssb, mod=4 level=0] ble_hs_hci_evt_acl_process(): conn_handle=1 pb=2 len=11 data=0x07 0x00 0x04 0x00 0x08 0x01 0x00 0x0b 0x00 0x03 0x28 
007966 [ts=62234360ssb, mod=4 level=0] rxed att command: read type req; conn=1 start_handle=0x0001 end_handle=0x000b
007966 [ts=62234360ssb, mod=4 level=0] txed att command: read type rsp; conn=1 length=7
007966 [ts=62234360ssb, mod=4 level=0] host tx hci data; handle=1 length=41
007966 [ts=62234360ssb, mod=4 level=0] ble_hs_hci_acl_tx(): 0x01 0x00 0x29 0x00 0x25 0x00 0x04 0x00 0x09 0x07 0x02 0x00 0x02 0x03 0x00 0x00 0x2a 0x04 0x00 0x02 0x05 0x00 0x01 0x2a 0x06 0x00 0x02 0x07 0x00 0x02 0x2a 0x08 0x00 0x08 0x09 0x00 0x03 0x2a 0x0a 0x00 0x02 0x0b 0x00 0x04 0x2a 
007966 [ts=62234360ssb, mod=4 level=0] Number of Completed Packets: num_handles=1
007966 [ts=62234360ssb, mod=4 level=0] handle:1 pkts:1
007969 [ts=62257796ssb, mod=4 level=0] ble_hs_hci_evt_acl_process(): conn_handle=1 pb=2 len=11 data=0x07 0x00 0x04 0x00 0x08 0x0c 0x00 0x0f 0x00 0x03 0x28 
007970 [ts=62265608ssb, mod=4 level=0] rxed att command: read type req; conn=1 start_handle=0x000c end_handle=0x000f
007970 [ts=62265608ssb, mod=4 level=0] txed att command: read type rsp; conn=1 length=7
007970 [ts=62265608ssb, mod=4 level=0] host tx hci data; handle=1 length=13
007970 [ts=62265608ssb, mod=4 level=0] ble_hs_hci_acl_tx(): 0x01 0x00 0x0d 0x00 0x09 0x00 0x04 0x00 0x09 0x07 0x0d 0x00 0x20 0x0e 0x00 0x05 0x2a 
007970 [ts=62265608ssb, mod=4 level=0] Number of Completed Packets: num_handles=1
007970 [ts=62265608ssb, mod=4 level=0] handle:1 pkts:1
007973 [ts=62289044ssb, mod=4 level=0] ble_hs_hci_evt_acl_process(): conn_handle=1 pb=2 len=11 data=0x07 0x00 0x04 0x00 0x08 0x1d 0x00 0x20 0x00 0x03 0x28 
007973 [ts=62289044ssb, mod=4 level=0] rxed att command: read type req; conn=1 start_handle=0x001d end_handle=0x0020
007973 [ts=62289044ssb, mod=4 level=0] txed att command: read type rsp; conn=1 length=21
007974 [ts=62296856ssb, mod=4 level=0] host tx hci data; handle=1 length=27
007974 [ts=62296856ssb, mod=4 level=0] ble_hs_hci_acl_tx(): 0x01 0x00 0x1b 0x00 0x17 0x00 0x04 0x00 0x09 0x15 0x1e 0x00 0x14 0x1f 0x00 0x48 0x7c 0x99 0x74 0x11 0x26 0x9e 0xae 0x01 0x4e 0xce 0xfb 0x28 0x78 0x2e 0xda 
007974 [ts=62296856ssb, mod=4 level=0] Number of Completed Packets: num_handles=1
007974 [ts=62296856ssb, mod=4 level=0] handle:1 pkts:1
007977 [ts=62320292ssb, mod=4 level=0] ble_hs_hci_evt_acl_process(): conn_handle=1 pb=2 len=7 data=0x03 0x00 0x04 0x00 0x0a 0x03 0x00 
007977 [ts=62320292ssb, mod=4 level=0] rxed att command: read req; conn=1 handle=0x0003
007977 [ts=62320292ssb, mod=4 level=0] host tx hci data; handle=1 length=19
007977 [ts=62320292ssb, mod=4 level=0] ble_hs_hci_acl_tx(): 0x01 0x00 0x13 0x00 0x0f 0x00 0x04 0x00 0x0b 0x6e 0x69 0x6d 0x62 0x6c 0x65 0x2d 0x62 0x6c 0x65 0x70 0x72 0x70 0x68 
007978 [ts=62328104ssb, mod=4 level=0] Number of Completed Packets: num_handles=1
007978 [ts=62328104ssb, mod=4 level=0] handle:1 pkts:1
007981 [ts=62351540ssb, mod=4 level=0] ble_hs_hci_evt_acl_process(): conn_handle=1 pb=2 len=7 data=0x03 0x00 0x04 0x00 0x0a 0x0b 0x00 
007981 [ts=62351540ssb, mod=4 level=0] rxed att command: read req; conn=1 handle=0x000b
007981 [ts=62351540ssb, mod=4 level=0] host tx hci data; handle=1 length=13
007981 [ts=62351540ssb, mod=4 level=0] ble_hs_hci_acl_tx(): 0x01 0x00 0x0d 0x00 0x09 0x00 0x04 0x00 0x0b 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 
007981 [ts=62351540ssb, mod=4 level=0] Number of Completed Packets: num_handles=1
007981 [ts=62351540ssb, mod=4 level=0] handle:1 pkts:1
007985 [ts=62382788ssb, mod=4 level=0] ble_hs_hci_evt_acl_process(): conn_handle=1 pb=2 len=9 data=0x05 0x00 0x04 0x00 0x04 0x0f 0x00 0x0f 0x00 
007985 [ts=62382788ssb, mod=4 level=0] rxed att command: find info req; conn=1 start_handle=0x000f end_handle=0x000f
007985 [ts=62382788ssb, mod=4 level=0] txed att command: find info rsp; conn=1 format=1
007985 [ts=62382788ssb, mod=4 level=0] host tx hci data; handle=1 length=10
007985 [ts=62382788ssb, mod=4 level=0] ble_hs_hci_acl_tx(): 0x01 0x00 0x0a 0x00 0x06 0x00 0x04 0x00 0x05 0x01 0x0f 0x00 0x02 0x29 
007985 [ts=62382788ssb, mod=4 level=0] Number of Completed Packets: num_handles=1
007985 [ts=62382788ssb, mod=4 level=0] handle:1 pkts:1
007989 [ts=62414036ssb, mod=4 level=0] ble_hs_hci_evt_acl_process(): conn_handle=1 pb=2 len=9 data=0x05 0x00 0x04 0x00 0x04 0x0f 0x00 0x0f 0x00 
007989 [ts=62414036ssb, mod=4 level=0] rxed att command: find info req; conn=1 start_handle=0x000f end_handle=0x000f
007989 [ts=62414036ssb, mod=4 level=0] txed att command: find info rsp; conn=1 format=1
007989 [ts=62414036ssb, mod=4 level=0] host tx hci data; handle=1 length=10
007989 [ts=62414036ssb, mod=4 level=0] ble_hs_hci_acl_tx(): 0x01 0x00 0x0a 0x00 0x06 0x00 0x04 0x00 0x05 0x01 0x0f 0x00 0x02 0x29 
007989 [ts=62414036ssb, mod=4 level=0] Number of Completed Packets: num_handles=1
007989 [ts=62414036ssb, mod=4 level=0] handle:1 pkts:1
007992 [ts=62437472ssb, mod=4 level=0] ble_hs_hci_evt_acl_process(): conn_handle=1 pb=2 len=9 data=0x05 0x00 0x04 0x00 0x04 0x20 0x00 0x20 0x00 
007993 [ts=62445284ssb, mod=4 level=0] rxed att command: find info req; conn=1 start_handle=0x0020 end_handle=0x0020
007993 [ts=62445284ssb, mod=4 level=0] txed att command: find info rsp; conn=1 format=1
007993 [ts=62445284ssb, mod=4 level=0] host tx hci data; handle=1 length=10
007993 [ts=62445284ssb, mod=4 level=0] ble_hs_hci_acl_tx(): 0x01 0x00 0x0a 0x00 0x06 0x00 0x04 0x00 0x05 0x01 0x20 0x00 0x02 0x29 
007993 [ts=62445284ssb, mod=4 level=0] Number of Completed Packets: num_handles=1
007993 [ts=62445284ssb, mod=4 level=0] handle:1 pkts:1
007996 [ts=62468720ssb, mod=4 level=0] ble_hs_hci_evt_acl_process(): conn_handle=1 pb=2 len=9 data=0x05 0x00 0x04 0x00 0x12 0x0f 0x00 0x02 0x00 
007996 [ts=62468720ssb, mod=4 level=0] rxed att command: write req; conn=1 handle=0x000f
007996 [ts=62468720ssb, mod=64 level=1] subscribe event; conn_handle=1 attr_handle=14 reason=1 prevn=0 curn=0 previ=0 curi=1
007997 [ts=62476532ssb, mod=4 level=0] txed att command: write rsp; conn=1 
007997 [ts=62476532ssb, mod=4 level=0] host tx hci data; handle=1 length=5
007997 [ts=62476532ssb, mod=4 level=0] ble_hs_hci_acl_tx(): 0x01 0x00 0x05 0x00 0x01 0x00 0x04 0x00 0x13 
007997 [ts=62476532ssb, mod=4 level=0] Number of Completed Packets: num_handles=1
007997 [ts=62476532ssb, mod=4 level=0] handle:1 pkts:1
008000 [ts=62499968ssb, mod=4 level=0] ble_hs_hci_evt_acl_process(): conn_handle=1 pb=2 len=9 data=0x05 0x00 0x04 0x00 0x12 0x0f 0x00 0x02 0x00 
008000 [ts=62499968ssb, mod=4 level=0] rxed att command: write req; conn=1 handle=0x000f
008000 [ts=62499968ssb, mod=4 level=0] txed att command: write rsp; conn=1 
008000 [ts=62499968ssb, mod=4 level=0] host tx hci data; handle=1 length=5
008000 [ts=62499968ssb, mod=4 level=0] ble_hs_hci_acl_tx(): 0x01 0x00 0x05 0x00 0x01 0x00 0x04 0x00 0x13 
008001 [ts=62507780ssb, mod=4 level=0] Number of Completed Packets: num_handles=1
008001 [ts=62507780ssb, mod=4 level=0] handle:1 pkts:1
008004 [ts=62531216ssb, mod=4 level=0] ble_hs_hci_evt_acl_process(): conn_handle=1 pb=2 len=9 data=0x05 0x00 0x04 0x00 0x12 0x20 0x00 0x01 0x00 
008004 [ts=62531216ssb, mod=4 level=0] rxed att command: write req; conn=1 handle=0x0020
008004 [ts=62531216ssb, mod=64 level=1] subscribe event; conn_handle=1 attr_handle=31 reason=1 prevn=0 curn=1 previ=0 curi=0
008004 [ts=62531216ssb, mod=4 level=0] txed att command: write rsp; conn=1 
008004 [ts=62531216ssb, mod=4 level=0] host tx hci data; handle=1 length=5
008004 [ts=62531216ssb, mod=4 level=0] ble_hs_hci_acl_tx(): 0x01 0x00 0x05 0x00 0x01 0x00 0x04 0x00 0x13 
008005 [ts=62539028ssb, mod=4 level=0] Number of Completed Packets: num_handles=1
008005 [ts=62539028ssb, mod=4 level=0] handle:1 pkts:1
008008 [ts=62562464ssb, mod=4 level=0] Number of Completed Packets: num_handles=1
008008 [ts=62562464ssb, mod=4 level=0] handle:1 pkts:1
008010 [ts=62578088ssb, mod=4 level=0] ble_hs_hci_evt_acl_process(): conn_handle=1 pb=2 len=15 data=0x0b 0x00 0x04 0x00 0x52 0x1f 0x00 0x00 0x00 0x00 0x00 0x00 0x01 0x00 0x00 
008010 [ts=62578088ssb, mod=4 level=0] rxed att command: write cmd; conn=1 handle=0x001f
2804636ssb, mod=4 level=0] ble_hs_hci_acl_tx(): 0x01 0x00 0x6c 0x00 0x68 0x00 0x04 0x00 0x1b 0x1f 0x00 0x69 0x72 0x6d 0x65 0x64 0xf5 0x66 0x61 0x63 0x74 0x69 0x76 0x65 0xf5 0x69 0x70 0x65 0x72 0x6d 0x61 0x6c 0x00 0x68 0x00 0x04 0x00 0x1b 0x1f 0x00 0x01 0x00 0x00 0xf4 0x00 0x01 0x00 0x00 0xbf 0x66 0x69 0x6d 0x61 0x67 0x65 0x73 0x9f 0xbf 0x64 0x73 0x6c 0x6f 0x74 0x00 0x67 0x76 0x65 0x72 0x73 0x69 0x6f 0x6e 0x65 0x30 0x2e 0x30 0x2e 0x30 0x64 0x68 0x61 0x73 0x68 0x58 0x20 0xd9 0xa7 0x35 0xac 0x58 0x2f 0x22 0x5f 0x8b 0x43 0xb7 0x1b 0x4d 0xae 0x29 0x50 0xdb 0x9a 0xa0 0x40 0x93 0x2b 0x6c 0xa1 0x17 0xf0 0x6d 0x6b 0xd7 0x2c 0x83 0xa2 0x68 0x62 0x6f 0x6f 0x74 0x61 0x62 0x6c 0x65 0xf5 0x67 0x70 0x65 0x6e 0x64 0x69 0x6e 0x67 0xf4 0x69 0x63 0x6f 0x6e 0x66 
008039 [ts=62804636ssb, mod=4 level=1] GATT procedure initiated: notify; att_handle=31
008039 [ts=62804636ssb, mod=4 level=0] txed att command: notify req; conn=1 handle=0x001f
008039 [ts=62804636ssb, mod=4 level=0] host tx hci data; handle=1 length=108
008039 [ts=65 0x6e 0x74 0xf4 0xff 0xbf 0x64 0x73 0x6c 0x6f 0x74 0x01 0x67 0x76 0x65 0x72 0x73 0x69 0x6f 0x6e 0x65 0x30 0x2e 0x30 0x2e 0x30 0x64 0x68 0x61 0x73 0x68 0x58 0x20 0x7a 0xd9 0x2c 0x45 0x61 0xa5 0x62 0x40 0xd2 0x33 0x4c 0xd6 0x54 0xee 0x50 0x44 0xe8 0x18 0x02 0xe5 0x6c 0x73 0x71 0x95 0x2b 0xe1 0x88 0xe7 0x28 0x65 0x8f 0xf1 0x68 0x62 0x6f 0x6f 0x74 0x61 0x62 0x6c 0x65 0xf4 0x67 0x70 0x65 0x6e 0x64 
008040 [ts=62812448ssb, mod=4 level=1] GATT procedure initiated: notify; att_handle=31
008040 [ts=62812448ssb, mod=4 level=0] txed att command: notify req; conn=1 handle=0x001f
008040 [ts=62812448ssb, mod=4 level=0] host tx hci data; handle=1 length=57
008040 [ts=62812448ssb, mod=4 level=0] ble_hs_hci_acl_tx(): 0x01 0x00 0x39 0x00 0x35 0x00 0x04 0x00 0x1b 0x1f 0x00 0x69 0x6e 0x67 0xf4 0x69 0x63 0x6f 0x6e 0x66 0x69 0x72 0x6d 0x65 0x64 0xf4 0x66 0x61 0x63 0x74 0x69 0x76 0x65 0xf4 0x69 0x70 0x65 0x72 0x6d 0x61 0x6e 0x65 0x6e 0x74 0xf4 0xff 0xff 0x6b 0x73 0x70 0x6c 0x69 0x74 0x53 0x74 0x61 0x74 0x75 0x73 0x02 0xff 
008043 [ts=62835884ssb, mod=4 level=0] Number of Completed Packets: num_handles=1
008043 [ts=62835884ssb, mod=4 level=0] handle:1 pkts:1
008044 [ts=62843696ssb, mod=4 level=0] Number of Completed Packets: num_handles=1
008045 [ts=62851508ssb, mod=4 level=0] handle:1 pkts:1
008048 [ts=62874944ssb, mod=4 level=0] Number of Completed Packets: num_handles=1
008048 [ts=62874944ssb, mod=4 level=0] handle:1 pkts:1
008052 [ts=62906192ssb, mod=4 level=0] ble_hs_hci_evt_acl_process(): conn_handle=1 pb=2 len=27 data=0x68 0x00 0x04 0x00 0x16 0x1f 0x00 0x00 0x00 0x02 0x00 0x00 0x66 0x00 0x01 0x01 0x01 0xa3 0x63 0x6f 0x66 0x66 0x00 0x64 0x64 0x61 0x74 
008052 [ts=62906192ssb, mod=4 level=0] ble_hs_hci_evt_acl_process(): conn_handle=1 pb=1 len=27 data=0x61 0x58 0x50 0x3c 0xb8 0xf3 0x96 0x24 0x00 0x00 0x00 0x20 0x00 0x00 0x00 0x64 0xf8 0x01 0x00 0x02 0x00 0x00 0x00 0x00 0x00 0x00 0x00 
008053 [ts=62914004ssb, mod=4 level=0] ble_hs_hci_evt_acl_process(): conn_handle=1 pb=1 len=27 data=0x01 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x01 0x20 0xb5 0x82 0x00 0x00 0x11 0x83 0x00 0x00 0x13 0x83 0x00 0x00 0x00 0x00 0x00 
00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x5b 0x94 
008054 [ts=62921816ssb, mod=4 level=0] rxed att command: prep write req; conn=1 handle=0x001f offset=0
008054 [ts=62921816ssb, mod=4 level=0] txed att command: prep write rsp; conn=1 handle=0x001f offset=0
008054 [ts=62921816ssb, mod=4 level=0] host tx hci data; handle=1 length=108
008054 [ts=62921816ssb, mod=4 level=0] ble_hs_hci_acl_tx(): 0x01 0x00 0x6c 0x00 0x68 0x00 0x04 0x00 0x17 0x1f 0x00 0x00 0x00 0x02 0x00 0x00 0x66 0x00 0x01 0x01 0x01 0xa3 0x63 0x6f 0x66 0x66 0x00 0x64 0x64 0x61 0x74 0x61 0x58 0x50 0x3c 0xb8 0xf3 0x96 0x24 0x00 0x00 0x00 0x20 0x00 0x00 0x00 0x64 0xf8 0x01 0x00 0x02 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x01 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x01 0x20 0xb5 0x82 0x00 0x00 0x11 0x83 0x00 0x00 0x13 0x83 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x5b 0x94 
008060 [ts=62968688ssb, mod=4 level=0] ble_hs_hci_evt_acl_process(): conn_handle=1 pb=2 len=20 data=0x10 0x00 0x04 0x00 0x16 0x1f 0x00 0x63 0x00 0x00 0x00 0x63 0x6c 0x65 0x6e 0x1a 0x00 0x01 0xf8 0xa8 
008060 [ts=62968688ssb, mod=4 level=0] rxed att command: prep write req; conn=1 handle=0x001f offset=99
008060 [ts=62968688ssb, mod=4 level=0] txed att command: prep write rsp; conn=1 handle=0x001f offset=99
008060 [ts=62968688ssb, mod=4 level=0] host tx hci data; handle=1 length=20
008060 [ts=62968688ssb, mod=4 level=0] ble_hs_hci_acl_tx(): 0x01 0x00 0x14 0x00 0x10 0x00 0x04 0x00 0x17 0x1f 0x00 0x63 0x00 0x00 0x00 0x63 0x6c 0x65 0x6e 0x1a 0x00 0x01 0xf8 0xa8 
008060 [ts=62968688ssb, mod=4 level=0] Number of Completed Packets: num_handles=1
008060 [ts=62968688ssb, mod=4 level=0] handle:1 pkts:1
008063 [ts=62992124ssb, mod=4 level=0] ble_hs_hci_evt_acl_process(): conn_handle=1 pb=2 len=6 data=0x02 0x00 0x04 0x00 0x18 0x01 
008064 [ts=63000000ssb, mod=4 level=0] rxed att command: exec write req; conn=1 flags=0x01
008064 [ts=63000000ssb, mod=4 level=0] host tx hci data; handle=1 length=5
008064 [ts=63000000ssb, mod=4 level=0] ble_hs_hci_acl_tx(): 0x01 0x00 0x05 0x00 0x01 0x00 0x04 0x00 0x19 
008064 [ts=63000000ssb, mod=4 level=0] Number of Completed Packets: num_handles=1
008064 [ts=63000000ssb, mod=4 level=0] handle:1 pkts:1
008079 [ts=63117180ssb, mod=4 level=1] GATT procedure initiated: notify; att_handle=31
008080 [ts=63124992ssb, mod=4 level=0] txed att command: notify req; conn=1 handle=0x001f
008080 [ts=63124992ssb, mod=4 level=0] host tx hci data; handle=1 length=27
008080 [ts=63124992ssb, mod=4 level=0] ble_hs_hci_acl_tx(): 0x01 0x00 0x1b 0x00 0x17 0x00 0x04 0x00 0x1b 0x1f 0x00 0x03 0x00 0x00 0x0c 0x00 0x01 0x01 0x01 0xbf 0x62 0x72 0x63 0x00 0x63 0x6f 0x66 0x66 0x18 0x50 0xff 
```

**Comment by andrzej-kaczmarek** [comment]
I think this is the same issue as #796 so please check fix for that issue.

**Comment by andrzej-kaczmarek** [comment]
Fixed by #810
